src/traefik.py

```diff
@@ -217,8 +217,9 @@ def remove_cas(self, uids: Iterable[Union[str, int]]):
         Calls update-ca-certificates once done.
         """
         for uid in uids:
-            ca_path = RECV_CA_TEMPLATE.substitute(rel_id=str(uid))
-            self._container.remove_path(ca_path)
+            files = self._container.list_files(CA_CERTS_PATH, pattern=f"receive-ca-cert-{uid}-*")
```

Comment:
I think files can be:
1. receive-ca-cert-UID
2. receive-ca-cert-UID-I

The pattern cannot have the last "-" or it'll miss format (1)

```suggestion
            files = self._container.list_files(CA_CERTS_PATH, pattern=f"receive-ca-cert-{uid}*")
```

Note that elsewhere I suggest we should only have one file name pattern rather than two, but either way this suggestion here would work

Reply:
(just noting - it isn't this PRs fault) I think there must not be test coverage for this deletion atm?

Reply:
Suggestion applied anyway.